cluster/gce/container-linux/configure-helper.sh

```diff
@@ -266,12 +266,12 @@ kind: Config
 users:
 - name: kubelet
   user:
-    client-certificate-data: ${KUBELET_CERT}
-    client-key-data: ${KUBELET_KEY}
+    token: ${KUBELET_TOKEN}
 clusters:
 - name: local
   cluster:
     certificate-authority-data: ${KUBELET_CA_CERT}
+    server: https://${KUBERNETES_MASTER_NAME}:443
```

Comment:
I think that's only true if we change the location of this file to not be the default location read by the kubelet. Right now it's ok that it's in the default location because the kubeconfig is neutered by not listing the address of the apiserver.